### ISSUE TITLE
subpath change-creds is not working when SYSTEM_ROOTURIPATH is set

### ISSUE THREAD

**Issue by terrytw** [Stirling-Tools/Stirling-PDF#1387]
Background: I set SYSTEM_ROOTURIPATH to /pdf, so stirling pdf is working at http://192.168.17.173:8080/pdf

When SECURITY_ENABLE_LOGIN is set to true, user will be prompted to change password during first login, which will redirect them to http://192.168.17.173/change-creds, but it doesn't work. It should be http://192.168.17.173/pdf/change-creds, but even manually type in http://192.168.17.173/pdf/change-creds user will be redirected to http://192.168.17.173/change-creds which shows:

![1](https://github.com/Stirling-Tools/Stirling-PDF/assets/24227628/f1337c2d-0dc1-436d-bfea-e2739a452d6f)

**EDIT:**
The following problem is stemmed from my browser, not stirling-PDF. 
~~Also appreantly PDF viewer is not working. None of these button work, and the stirling PDF logo redirects to root path http://192.168.17.173:8080 instead of the subpath /pdf~~
~~![2](https://github.com/Stirling-Tools/Stirling-PDF/assets/24227628/82059075-97c0-4a8d-ac5d-ca093ece8508)~~

~~Honestly a lot of the tools doesn't work, I guess they reference the pdf viewer in order to function, like the rotation tools, page preview just doesn't show up.~~

**Comment by Ludy87** [comment]
Can you please tell us the version you are using, I just tested it and it works as expected.

**Comment by Ludy87** [comment]
ah, you mean when you send the document the error occurs.

**Comment by terrytw** [comment]
1. I just pulled docker image with the latest tag, it seems to be from 4 days ago.
Here is the docker run command:
```
docker run -d \
  --restart unless-stopped \
  --name Stirling-PDF \
  --hostname Stirling-PDF \
  --net custom --ip 172.18.0.6 \
  -e PUID=*** \
  -e PGID=*** \
  -e TZ=America/*** \
  -e SYSTEM_ROOTURIPATH=/pdf \
  -e DOCKER_ENABLE_SECURITY=true \
  -e SECURITY_ENABLE_LOGIN=true \
  -e INSTALL_BOOK_AND_ADVANCED_HTML_OPS=true \
  -e LANGS=en_GB \
  -p 8080:8080 \
  -v /**/Stirling-PDF/data:/usr/share/tessdata \
  -v /**/Stirling-PDF/configs:/configs \
  -v /**/Stirling-PDF/logs:/logs \
  frooodle/s-pdf:latest
```
2. I observed 2 different problems, 1 is that /change-creds is not correctly redirected to /pdf/change-creds, 2 is that pdf viewer is not working.
3. Upon further investigation, problem 2 seems to be stemmed from my browser, after switching to a brand new chrome installation in a clean VM, I can use the PDF viewer and rotate tools.
4. Problem 1 persists even with clean installation of chrome.

**Comment by Frooodle** [comment]
fixed in 0.25.2!
